### PR TITLE
Scala 2.12.14 / 2.13.6

### PR DESCRIPTION
**Pull request by mkurz**
Similiar to #6426 and #6291

### CODE REVIEW

**Comment by mkurz** [comment]
~Waiting for https://github.com/typelevel/kind-projector/pull/192...~ :eyes: Merged.

**Comment by mkurz** [comment]
Similiar pull requests for other projects:
* https://github.com/sbt/launcher/pull/97
* https://github.com/sbt/zinc/pull/980
* https://github.com/sbt/io/pull/318
* https://github.com/sbt/librarymanagement/pull/381

File: project/Dependencies.scala [hunk 1]
```diff
@@ -116,5 +116,5 @@ object Dependencies {
 
   val hedgehog = "qa.hedgehog" %% "hedgehog-sbt" % "0.6.1"
   val disruptor = "com.lmax" % "disruptor" % "3.4.2"
-  val kindProjector = ("org.typelevel" % "kind-projector" % "0.11.3").cross(CrossVersion.full)
+  val kindProjector = ("org.typelevel" % "kind-projector" % "0.13.0").cross(CrossVersion.full)
```
Comment (mkurz) [inline project/Dependencies.scala:119]: Had to upgrade `kind-projector`, but that should be ok:
https://github.com/typelevel/kind-projector/compare/v0.11.3...v0.13.0

File: launcher-package/build.sbt [hunk 2]
```diff
@@ -26,15 +26,15 @@ lazy val sbtVersionToRelease = sys.props.getOrElse("sbt.build.version", sys.env.
...
+lazy val scala212 = "2.12.14"
 lazy val scala210Jline = "org.scala-lang" % "jline" % scala210
 lazy val jansi = {
   if (sbtVersionToRelease startsWith "1.") "org.fusesource.jansi" % "jansi" % "1.12"
   else "org.fusesource.jansi" % "jansi" % "1.4"
 }
 lazy val scala212Compiler = "org.scala-lang" % "scala-compiler" % scala212
 lazy val scala212Jline = "jline" % "jline" % "2.14.6"
-// use the scala-xml version used by the compiler not the latest: https://github.com/scala/scala/blob/v2.12.10/versions.properties#L22
+// use the scala-xml version used by the compiler not the latest: https://github.com/scala/scala/blob/v2.12.14/versions.properties#L21
```
Comment (mkurz) [inline launcher-package/build.sbt:37]: `scala-xml` version did _not_ change :sweat_smile:

**Review by eed3si9n — DISMISSED** [review]
@mkurz Thanks for the contribution!

**Comment by eed3si9n** [comment]
I guess the question is how safe this is to release without RC cycle. Sometimes Scala 2.12.x includes extra check that starts displaying new warnings. Have you tried using locally baked sbt to a build to see if you see anything weird?

**Comment by mkurz** [comment]
> Have you tried using locally baked sbt to a build to see if you see anything weird?

No, but I can do that (later today)

**Comment by mkurz** [comment]
To be able to publish local I needed to upgrade semanticdb as well: https://github.com/scalameta/scalameta/releases

**Comment by mkurz** [comment]
@eed3si9n I did some local testing by `publishLocal` a `1.5.3-SNAPSHOT`. Everything went smoothly. Using that snapshot, I published locally an inhouse sbt-plugin and afterward used that sbt plugin in different projects, which itself also made use of the custom sbt`1.5.3-SNAPSHOT`.
Everything worked and compiled, without any weird warnings. I also tried a Play Framework project, it ran normally, no warnings.

For me this LGTM.

**Comment by eed3si9n** [comment]
Thanks for testing.

**Pull request merged**